mne/io/base.py

```diff
@@ -888,8 +935,8 @@ def get_data(self, picks=None, start=0, stop=None,
             data, times = self[picks, start:stop]
 
         if return_times:
-            return data, times
-        return data
+            return data * ch_factors[:, np.newaxis], times
+        return data * ch_factors[:, np.newaxis]
```

Comment:
this code will now generate a new copy of the data on return even if no specific scaling is required. Given how much the IO code has been optimized over the years I think this is not OK. Basically no need to multiply by np.ones if ch_factors has not been set.